README.md

```diff
@@ -1,3 +1,59 @@
+# Math Magicians
+
+a React app calculator
+
+## Built With
+
+- Major languages - React.
+- Frameworks - 
+- Technologies used - Visual Studio Code, Git and Github, Chrome (Browser and it's Dev Tools), Accessibility Tools (Wave Evaluation ool and Web Developer)
+- Testing: Linter.
+
+## Live Demo
+
+N/A
+
+### Prerequisites 
+  - Local Machine, Git, Text Editor (VS Code preferably)
+
+### Setup 
+  - Checkout the project repository here [Link](https://github.com/Khaled-AlKhateeb/math-magicians.git)
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] It would be nice if you could give instructions steps on how to get the project up and running:

Steps like:
- `git clone https://github.com/Khaled-AlKhateeb/math-magicians.git`
- `cd math-magicians`

Reply:
yeah its at the end of the readme.md file.
thank you for the review